src/http-server/routes/github/index.js

```diff
@@ -1,43 +1,24 @@
 const GithubPush = require('windbreaker-service-util/models/events/webhook/github/GithubPush')
-const WebhookType = require('~/src/models/WebhookType')
-const dao = require('~/src/dao')
+const Installation = require('windbreaker-service-util/models/events/webhook/github/Installation')
 const logger = require('~/src/logging').logger(module)
 const producer = require('~/src/producer')
-const getWebhookMiddleware = require('~/src/http-server/util/getWebhookMiddleware')
+const getWebhookMiddleware = require('~/src/http-server/util/github/getWebhookMiddleware')
 
 const eventToModelType = {
-  push: GithubPush
+  push: GithubPush,
+  installation: Installation
 }
 
 const EVENT_HEADER = 'x-github-event'
 
-async function _isValidWebhook (id) {
-  try {
-    let result = await dao.isValidWebhook(id, WebhookType.GITHUB)
-    return result
-  } catch (err) {
-    logger.error(`Error validating webhook with id "${id}"`)
-    return false
-  }
-}
-
 module.exports = {
   method: 'POST',
-  path: '/github/:id',
+  path: '/github',
   middleware: getWebhookMiddleware(),
   async handler (ctx) {
```

Comment:
I just removed the :id code from here and tried to leave most of the rest of the :id related helpers/tests intact when possible. It seems like we'll need some kind of alternative approach for bitbucket, I just don't know what it'll be yet. Once we figure out the approach, we should try to consolidate as much as possible.